LandMark/Feature/Home/Pin/Data/Location/LocationManager.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  LocationManager.swift
+//  LandMark
+//
+//  Created by Lucas Migge on 31/10/24.
+//
+
+import CoreLocation
+
+protocol LocationManagerProtocol {
+    func getCurrentUserLocation() -> LocationCoordinate
+}
+
+class LocationManager: NSObject, CLLocationManagerDelegate, ObservableObject, LocationManagerProtocol {
```

Comment:
Sugestão pra deixar isso mais significativo e abrir espaço pra implementações diferentes:
```suggestion
protocol LocationManager {
    func getCurrentUserLocation() -> LocationCoordinate
}

class CoreLocationLocationManager: NSObject, CLLocationManagerDelegate, ObservableObject, LocationManagerProtocol {
```